.travis.yml

```diff
@@ -9,17 +9,11 @@ matrix:
       python: "3.5"
       env: ATOM_CHANNEL=beta
     - os: osx
-      language: generic
-      before_install:
-        - brew update
-        # Need a way to check if it's installed already or not...
-        - brew install python
```

Comment:
Me either, I found that somewhere online at some point and have been copying it around since 😛.